server/db/repositories/queries/find.go

```diff
@@ -1,21 +1,23 @@
 package queries
 
-const (
+var (
```

Comment:
Maybe instead of strings use functions? e.g.:
```golang
var findAll = `
    FOR i IN %s
        LIMIT @offset, @count
        FILTER @` + ParamPageCursor + ` != NULL ? i.created_at < @` + ParamPageCursor + ` : TRUE == TRUE
        LIMIT @` + ParamPageCount + `
        RETURN i
`

func FindAll(collection string) string {
    return fmt.Sprintf(findAll, collection)
}
```
So, usage will be
```golang
cursor, err := repo.collection.Database().Query(
    ctx,
    queries.FindAll(repo.collection.Name())
    map[string]interface{}{
        "offset": query.Pagination.Size * (query.Pagination.Page - 1),
        "count":  query.Pagination.Size,
    },
    params,
)
```

In my opinion,  this 
```golang
queries.FindAll(repo.collection.Name())
```
is more readable than this
```golang
fmt.Sprintf(queries.FindAll, repo.collection.Name()),
```